Makefile

```diff
@@ -77,6 +77,11 @@ fips-ready: PDFFILE=wolfSSL-FIPS-Ready.pdf
 fips-ready: build
 	@$(DOCKER_CMD)
 
+.PHONY: tuning
```

Comment:
You must also add `tuning` to the "all" section at top.

Reply:
I also added porting as well to the top as it was not included